### DISCUSSION TITLE
posts/my-new-post/

### DISCUSSION THREAD

**Discussion by giscus** [jenhaoyang/jenhaoyang.github.io#2] [Announcements]
# posts/my-new-post/

1. 安裝CUDA、cuDNN、Tensorrt Visual Studio 2019(注意 先安裝visual studio 再安裝CUDA，因為CUDA包含給visual studio使用的原件) cmake: 3.24.0 CUDA: 11.4 cuDNN: 8.2 Tensorrt:8.2.3.0

https://jenhaoyang.github.io/posts/my-new-post/

**Comment by jenhaoyang** [comment]
Hello 留言
